src/MongoDB.Driver/IMongoCollectionExtensions.cs

```diff
@@ -163,6 +163,7 @@ public static Task<long> CountAsync<TDocument>(this IMongoCollection<TDocument>
 
         /// <summary>
         /// Counts the number of documents in the collection.
+        /// For a fast estimate of the total documents in a collection see <see cref="IMongoCollection{TDocument}.EstimatedDocumentCount"/>.
```

Comment:
The type parameters are required for the compile time checks to pass on the docstring